core/widgetdiv.js

```diff
@@ -147,6 +146,5 @@ Blockly.WidgetDiv.position = function(anchorX, anchorY, windowSize,
   }
   Blockly.WidgetDiv.DIV.style.left = anchorX + 'px';
   Blockly.WidgetDiv.DIV.style.top = anchorY + 'px';
-  Blockly.WidgetDiv.DIV.style.height =
-      (windowSize.height - anchorY + scrollOffset.y) + 'px';
+  Blockly.WidgetDiv.DIV.style.height = windowSize.height - anchorY + 'px';
```

Comment:
If this line is not synchronised with what's on master there will be issues with multiple workspaces taking a vertical size larger than the browser viewport.